Rakefile

```diff
@@ -0,0 +1,12 @@
+desc "Build docs and install to Xcode"
```

Comment:
Yes, I used `rake` because they did.
I considered a simple bash script ... that's how we did it in Mobile Data ... but this looked just as easy.